metplotpy/plots/polar_plot/polar_ice.yaml

```
@@ -1,4 +1,6 @@
-input_file: "~/grid_stat_north_000000L_20210305_120000V_pairs.nc"
+input_file: "path-to-data/grid_stat_north_000000L_20210305_120000V_pairs.nc"
 forecast_netcdf_var_name: "FCST_ice_coverage_SURFACE_FULL"
 obs_netcdf_var_name: "OBS_ice_coverage_SURFACE_FULL"
 diff_netcdf_var_name: "DIFF_ice_coverage_SURFACE_ice_coverage_SURFACE_FULL"
+log_level: info
+log_filename: "path-to-log/log.txt"
```

```suggestion
log_filename: ./polar_ice_log.txt
```
Suggesting this change for consistency with the other files.